LenovoLegionToolkit.Lib.Automation/Steps/RunAutomationStep.cs

```diff
@@ -10,11 +10,17 @@ public class RunAutomationStep : IAutomationStep
 
     public string? ScriptArguments { get; }
 
+    public bool? RunSilently { get; }
```

Comment:
Avoid optional bool at all cost.

Reply:
okay, null-judgement will be moved to step control (cuz `CheckBox.IsChecked` returns an optional bool).